### PR TITLE
chore: use slim ruby image

### PR DESCRIPTION
**Pull request by mc-jones**
co-authored with: @ovasdi 

### Problem
Rosalind's docker image is quite large at `2.4 GB` in size. This results in occasional timeouts during deploy steps when trying to pull it from `ecr`, [for example.](https://app.circleci.com/pipelines/github/artsy/rosalind/2172/workflows/b4528c50-c33a-422f-880a-0c099cd2f2df/jobs/2424?invite=true#step-106-12) 

### Solution
Optimize the docker image by utilizing the official `slim` docker image. Using the slim image cuts the size by `~400 mb`, or, about 20%. This should be sufficient for limiting timeouts, as other images of this size don't see the same issue. 

```
$ docker image ls
REPOSITORY              TAG       IMAGE ID       CREATED        SIZE
rosalind_slim           latest    9b9fa8dd1383   5 hours ago    1.93GB
rosalind                latest    9e482fe25905   22 hours ago   2.4GB
```

### Considerations
We've moved away from utilizing the shared [artsy docker](https://github.com/artsy/docker-images/tree/main/ruby) image in order to decouple the work sliming this image from another open question about how to address an edge case of `google-chrome-stable` not being compatible with `arm` architectures (ticket to track this issue to be created). 

[PHIRE-24]

[PHIRE-24]: https://artsyproduct.atlassian.net/browse/PHIRE-24?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -36,4 +36,4 @@
 
 # only look for driver updates if the previous check was
 # longer ago than a specified number of seconds
-Webdrivers.cache_time = 86_400
+# Webdrivers.cache_time = 86_400
```
Comment (mc-jones) [inline spec/rails_helper.rb:39]: @ovasdi - should we just delete this? Wasn't sure if we just wanted to comment while working locally.

**Review by anandaroop — APPROVED** [review]
If Circle is happy, I'm happy. 

Thanks for taking this on 🙏🏽

File: spec/rails_helper.rb (hunk 1 above)
Comment (ovasdi) [reply spec/rails_helper.rb:39]: Let's remove it seeing as [Webdrivers was removed](https://github.com/artsy/rosalind/pull/739/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fL412).

Comment (mc-jones) [reply spec/rails_helper.rb]: Done in [5c9ae9c](https://github.com/artsy/rosalind/pull/739/commits/5c9ae9cb9107e9e449d54622e44f2f355fdc7e57)

**Pull request merged**